client/src/pages/FavouritesPage.js

```diff
@@ -0,0 +1,21 @@
+import React from "react";
```

Comment:
I think with the latest react version -import React from 'react'- is done by default / consider removing this